guacamole-ext/src/main/resources/org/apache/guacamole/protocols/rdp.json

```diff
@@ -195,6 +195,10 @@
                 {
                     "name"  : "static-channels",
                     "type"  : "TEXT"
+                },
+                {
+                    "name"  : "timezone",
+                    "type"  : "TIMEZONE"
```

Comment:
I don't recall if this was mentioned before, but the `TIMEZONE` field type uses the Java IDs for its values:

https://github.com/apache/guacamole-client/blob/e02dcf8c84f0f7f2f9c866b0258bde0551c847b9/guacamole/src/main/webapp/app/form/controllers/timeZoneFieldController.js#L21-L25

This won't match the IANA values required by the `timezone` parameters. The recent changes from #403 use a `TEXT` field for this, however if you'd like to provide convenient drop-downs, etc., a new field type could be defined that uses the IANA values (`IANA_TIMEZONE`?)

Beware also that the #403 changes added this in a different section, so if you feel that this should be in `device-redirection`, the other parameter definition will need to be deleted.

Reply:
Or ... do we know that the Java IDs match?

Reply:
According to this post that I found:
https://stackoverflow.com/questions/44582795/java-8-tz-database-time-zones
...the database shipped with Java is at least based on the same one as the IANA/Olson/Tz Database (tzdb - https://en.wikipedia.org/wiki/Tz_database).  My guess is that it's a reasonably safe assumption that these will match up - or, at the very least, that one is going to be just as out-of-date as another.

I'm happy to implement an `IANA_TIMEZONE` text field, as you suggest, if you think that's the right way, but my gut feeling is that the net value of implementing and maintaining it separately from the other timezones already defined within the code is going to be small, and that, by and large, the timezones already defined within the existing timezone field are going to work fine.

Reply:
Ah, OK then. If we're reasonably sure that they line up, I'm happy as long as the `TIMEZONE` field's documentation is updated to match.

Reply:
I did a quick search of all the time zones in my zoneinfo database on my Linux system, and absolutely all time zones currently defined within the field match **except one**: `Canada/East-Saskatchewan`.

This is apparently a rare case of a time zone identifier being removed from the database:

https://stackoverflow.com/questions/47618861/error-invalid-value-for-parameter-timezone-canada-east-saskatchewan/47625517#47625517

So ... I think we are indeed safe, but should probably remove `Canada/East-Saskatchewan`.

Reply:
OK, I've opened #404 (PR not found!) to hopefully help with this and keep scope narrow.